src/Network/Network/Properties/Resources.Designer.cs

```diff
@@ -267,6 +267,15 @@ internal class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Cannot have both &apos;Commission&apos; and &apos;Decomission&apos; flags set.
+        /// </summary>
+        internal static string CommissioningStateConflict {
+            get {
+                return ResourceManager.GetString("CommissioningStateConflict", resourceCulture);
```

Comment:
Did you not check in the change in `Resources.resx`?

Reply:
you're right, fixed